pkgs/development/libraries/libticonv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchurl
+, pkgconfig
```

Comment:
```suggestion
, pkg-config
```

Reply:
Not strictly necessary IMO since they're literally equal.

Reply:
`pkg-config` is preferred over `pkgconfig`